docs_src/microservices/support/notifications/Ch-AlertsNotifications.md

```diff
@@ -53,114 +51,99 @@ senders send out the notifications to the subscribed receivers.
 
 ### Workflow
 
-#### Normal Notifications
-When a client requests a notification to be sent with "NORMAL" status, the notification is queued up (batched with other normal notifications).  The Message Scheduler, under a configurable interval, calls on the Distribution Coordinator to send all normal notifications to their receivers.  When a normal notification fails to be sent, it is retried a configurable number of times (resend limit).  After exceeding the resent tries, the notification is elevated to "CRITICAL" status and then sent through the critical notifications workflow below.
+#### Normal/Minor Notifications
+When a client requests a notification to be sent with "NORMAL" or "MINOR" status, 
+the notification is immediately sent to its receivers via the Distribution Coordinator, 
+and the status is updated to "PROCESSED".
 
 #### Critical Notifications
-Notifications that are sent with "CRITICAL" status, or notifications that have failed to send via the NORMAL workflow are immediately sent to their receivers via the Distribution Coordinator.  If a critical notification fails to send for a specified number of retries, the service escalates the notification in order to notify an escalation subscriber of the failure to notify the receiver.
+Notifications with "CRITICAL" status are also sent immediately.
+When encountering any error during sending critical notification, 
+an individual resend task is scheduled, and each transmission record persists.
+After exceeding the configurable limit (resend limit), the service escalates the notification 
+and create a new notification to notify particular receivers of the escalation subscription (name = "ESCALATION") of the failure.
+
+!!! edgey "Edgex 2.0" 
+    For EdgeX 2.0, all notifications are processed immediately. The resend feature is only provided for critical notifications.
+    The **resendLimit** and **resendInterval** properties can be defined in each subscription. 
+    If the properties are not provided, use the default values in the configuration properties.
 
 ## Data Model
 
 ![image](EdgeX_SupportingServicesNotificationsModel.png)
 
```

Comment:
Please refer to https://github.com/edgexfoundry/edgex-docs/blob/3a731fecc851abb2bb60fdd9d46fea3b2abff36e/docs_src/microservices/support/scheduler/Ch-Scheduler.md

Add a statement before the diagram like
**The latest developed data model will be updated in the Swagger API document (with link).**

After the diagram, add another statement:
**This diagram is drawn by diagram.net (https://app.diagrams.net/), and the source file is here (link to EdgeX_SupportingServicesNotificationsModel.xml )**

Reply:
Updated.